helm-chart/values.yaml

```diff
@@ -7,7 +7,7 @@ replicaCount: 1
 image:
   repository: supervisions/itop
   pullPolicy: Always
-  tag: 3.0
+  tag: "3.0"
```

Comment:
I also made this chage because helm interprete 3.0 as integer and translate it as `supervisions/itop:3`. With that quote helm translate by `supervisions/itop:3.0.X`